.buildkite/scripts/release-manager.sh

```diff
@@ -43,9 +43,9 @@ run_release_manager() {
     fi
     docker run --rm \
     --name release-manager \
-    -e VAULT_ADDR_SECRET \
-    -e VAULT_ROLE_ID_SECRET \
-    -e VAULT_SECRET_ID_SECRET \
+    -e VAULT_ADDR="${VAULT_ADDR_SECRET}" \
+    -e VAULT_ROLE_ID="${VAULT_ROLE_ID_SECRET}" \
+    -e VAULT_SECRET_ID="${VAULT_SECRET}" \
```

Comment:
The env var for the secret id is named differently 
https://github.com/elastic/fleet-server/blob/e1a2d076fe4eec9f38665cd32f7c69b640f94d63/.buildkite/hooks/pre-command#L77

```suggestion
    -e VAULT_SECRET_ID="${VAULT_SECRET_ID_SECRET}" \
```